packages/flutter_tools/test/commands.shard/hermetic/build_web_test.dart

```diff
@@ -111,7 +111,6 @@ void main() {
         'DartObfuscation': 'false',
         'TrackWidgetCreation': 'false',
         'TreeShakeIcons': 'false',
-        'baseHref': null,
```

Comment:
There's no reason to pass in a null value into this map since the absence of the key and a null value are handled the same:
https://github.com/flutter/flutter/blob/c584a5ead46c439d27a6f24466df5e4fe0c49476/packages/flutter_tools/lib/src/build_system/targets/web.dart#L390-L391